src/bson.app.src

```diff
@@ -1,7 +1,7 @@
 %% ex: ts=4 sw=4 noexpandtab syntax=erlang
 {application, bson, [
   {description, "BSON are JSON-like objects with a standard binary serialization. See bsonspec.org"},
-  {vsn, "v0.2.2"},
+  {vsn, "v0.2.4"},
```

Comment:
@comtihon I bumped the version to 0.2.4, since the most recent tag as of this writing is 0.2.3.